tests/cypress/integration/actions_tasks3/case_112_tus_upload.js

```diff
@@ -35,8 +34,10 @@ context('Create task with tus file', () => {
 
     describe(`Testing "${labelName}"`, () => {
         it('Create a task with 5mb upload chunk size', () => {
-            cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName,
-                null, advancedConfigurationParams);
+            const win = cy.state('window');
+            win.cvat.config.uploadChunkSize = 5;
+            cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName)
+                .then(() => { win.cvat.config.uploadChunkSize = 100; });
```

Comment:
Maybe better to use ``finally`` instead of ``then``?

Reply:
Yes, it is better choice, if task creation fails, upload size won't go back. But `cy.createAnnotaitionTask` do not return Promise, its cypress object that has no `.catch` or `.finally`. They have some reasons to avoid adding error handling described here https://docs.cypress.io/guides/core-concepts/conditional-testing#Error-Recovery
Maybe for our case its not so necessary because any other test data does not exceed 5mb